splice_machine/src/main/java/com/splicemachine/pipeline/IndexWriteHandler.java

```diff
@@ -109,7 +102,13 @@ public boolean route(KVPair mutation,WriteContext ctx) {
                     mutation = getBaseUpdateMutation(mutation);
                     return createIndexRecord(mutation, ctx, delete);
                 }
-                return true; // No index columns modifies ignore...
+                return true; // No index columns modified, ignore...
+            case BLIND_UPDATE:
+                if(transformer.areIndexKeysModified(mutation)) {
```

Comment:
I think this function has to be updated, or a different function is needed for BLIND_UPDATES. The logic here has to be: if any indexed column is modified, we process the update. 
Instead of: only when all indexed columns are modified, we process the update.

Reply:
I think a test case for this would be something like:
```
create table a (i int primary key);
create table b (i int references a(i) on delete set null, j int);
create index bi on b (i,j);
insert into a values 1;
insert into b values (1,1);
delete from a;
select * from b --splice-properties index=bi
;
```

Reply:
Thanks, I had that in my notes to address but I forgot to do it. I will push a fix for it plus a test.